src/jit/morph.cpp

```diff
@@ -14057,6 +14057,12 @@ GenTree* Compiler::fgMorphModToSubMulDiv(GenTreeOp* tree)
         denominator = fgMakeMultiUse(&tree->gtOp2);
     }
 
+    if (lvaLocalVarRefCounted)
```

Comment:
In this particular case, we caught an assert where the refcounts for the numerator went below zero because we did not have a corresponding increment for a decrement done at morph.cpp line 13186.